### PR TITLE
Pure 0.67 update

### PR DESCRIPTION
**Pull request by agraef**
Update pure to the latest release (cf. https://github.com/agraef/pure-lang/releases/tag/pure-0.67), including pure-docs 0.67 which was released along with it. Tested all functionality and included documentation, ran `make check` on the interpreter, all fine. Pure runtime library is still the same (no API or ABI changes), so all addon modules in the pure group continue to work.

Please note that Pure is hosted on Github now, which called for some changes in the Portfiles. Almost all addon releases are still on Bitbucket, though, hence the pure port group must remain as is for now.

One important new feature in this release is on-the-fly syntax checking of Pure scripts in Emacs. Thus flycheck-pure.el was added to the pure-mode.el subport, but this elisp package isn't byte-compiled right now, because this would require the Emacs flycheck package which isn't available in MacPorts. (Emacs users will typically install flycheck from MELPA anyway, and then flycheck-pure.el can be used, which is why it's included.)

###### Tested on

macOS 10.12.6 16G1212
Xcode 9.2 9C40b

### CODE REVIEW

File: lang/pure/Portfile [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 PortSystem              1.0
-PortGroup               bitbucket 1.0
+PortGroup               github 1.0
 
 # Keep the versions of pure and pure-docs in sync.
-bitbucket.setup         purelang pure-lang 0.66 pure-
-name                    pure
+name                            pure
+github.setup                    agraef pure-lang 0.67 ${name}-
```
Comment (ryandesign) [inline lang/pure/Portfile]: Could you fix the alignment of these two lines?

File: lang/pure/Portfile [hunk 2]
```diff
@@ -28,18 +28,18 @@ long_description        Pure is a functional programming language based on \
...
                         and the examples are BSD-licensed.
 
-bitbucket.tarball_from  downloads
+github.tarball_from     releases
 distname                ${name}-${version}
 
-checksums               rmd160  a3ad0dccdb71eab3aa8ee15d3a04f5e3111208a1 \
-                        sha256  42df6832476e8bee3a7ca179671284c1edd7bc82b71062fa0de62fd2117ee676
+checksums               rmd160  68f52aafe4b378fbafe7bb43fdee5612f1acd666 \
+                        sha256  442de3ccc98f55e7c04a7470771bd8d92e49a43f897964cc5b191c6b7a2b9dc2
```
Comment (ryandesign) [inline lang/pure/Portfile]: Could you add the `size` parameter (the filesize in bytes) to the `checksums` here and in the other port?

File: lang/pure/Portfile (hunk 1 above)
Comment (agraef) [reply lang/pure/Portfile]: Done.

File: lang/pure/Portfile (hunk 2 above)
Comment (agraef) [reply lang/pure/Portfile]: Done.

**Comment by agraef** [comment]
All done and squashed again.

**Comment by mojca** [comment]
Thank you

**Pull request merged**